identity-iota/src/credential/errors.rs

```diff
@@ -0,0 +1,138 @@
+// Copyright 2020-2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use std::collections::BTreeMap;
+use std::fmt::Display;
+
+use identity_core::common::OneOrMany;
+
+use crate::did::IotaDIDUrl;
+
+#[derive(Debug, thiserror::Error)]
+#[non_exhaustive]
+/// An error associated with validating credentials and presentations.
+pub enum ValidationError {
+  /// Indicates that the expiration date of the credential is not considered valid.
+  #[error("the expiration date does not satisfy the validation criterea")]
```

Comment:
"The credential has been expired as the expiration date has been passed" I feel like the current error is a bit too technically written. As a developer I understand that this should be caught and the application might want to present a more user friendly error message, but as a developer I also just clearly want to know what is up. Maybe hinting a bit more at the cause could help.

Reply:
Updated the description.